frontend/public/components/_nav-title.scss

```
@@ -24,6 +24,15 @@
     padding-left: $grid-gutter-width;
     padding-right: $grid-gutter-width;
   }
+  &--row {
+    @media (min-width: $screen-sm) {
+      display: flex;
+      flex-direction: row;
+    }
+    @media (max-width: $screen-sm) {
```

```suggestion
    @media (max-width: $screen-xs-max) {
```
as that is [767px](https://github.com/twbs/bootstrap/blob/v3.4.1/less/variables.less#L317) instead of [768px](https://github.com/twbs/bootstrap/blob/v3.4.1/less/variables.less#L297).  Otherwise you'll get your bottom margin at 768px, too.  And put this rule first not only for alpha but also for mobile-first.

---

In other words, move lines 32-34 to line 28.

---

@rhamilto we only want `30px margin` when we don't have the button and title in the same line. We show them in the same line in size > `$screen-xs (480px)` so adding `margin` at size < `$screen-xs-max (767px)` leaves us with unnecessary margin in the range of `480px to 767px`.
Which causes issues similar to this: https://github.com/openshift/console/pull/5080#discussion_r410226261